### PR TITLE
Remove twitter and stack overflow

### PR DESCRIPTION
**Pull request by adiroiban**
I don't think that any of the active developers are active on those platforms

so better to remove them and focus on mailing list

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -575,6 +576,7 @@ <h4>Report a
                   </h4>
                 </div>
               </div>
+
```
Comment (graingert) [inline index.html:579]:
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -587,18 +589,7 @@ <h4>Ask on
...
-                </div>
-                <div>
-                  <h4>Join the
-                    <a href="https://mail.python.org/mailman3/lists/twisted.python.org/">
-                      discussion list
-                    </a>
-                  </h4>
-                </div>
-              </div>
+
```
Comment (graingert) [inline index.html:592]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -550,14 +550,15 @@ <h4>Supported Python</h4>
                   </p>
                 </div>
               </div>
+
```
Comment (graingert) [inline index.html:553]:
```suggestion
```

**Comment by adiroiban** [comment]
I have removed stack overflow  to try to get the UI alligned  ... but I agree that is best to remove it in a separate PR

thanks for the review

image

File: index.html (hunk 3 above)
Comment (adiroiban) [reply index.html:553]: I prefer to have these extra newlines to make it easier to see where the tags start and end

**Review by graingert — APPROVED** [review]

**Pull request merged**